frontend/device/generic/device.lua

```diff
@@ -134,6 +141,9 @@ function Device:onPowerEvent(ev)
         DEBUG("Resuming...")
         local UIManager = require("ui/uimanager")
         UIManager:unschedule(self.suspend)
+        if self.wifi_was_on and G_reader_settings:readSetting("auto_restore_wifi") then
```

Comment:
I suggest we make it a default on feature to keep it consistent with other systems.